.github/workflows/ci.yaml

```diff
@@ -15,16 +15,9 @@
 name: CI
 
 on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
   push:
     branches:
-      - main
```

Comment:
Why remove main? We also want to trigger e2e tests on pushes to main branch

Reply:
k

Reply:
Done